packages/react-menu/src/tmp-icons.stories.tsx

```diff
@@ -84,3 +84,8 @@ export const EditIcon = renderIcon(props => (
     <path d="M2048 335q0 66-25 128t-73 110L633 1890 0 2048l158-633L1475 98q48-48 110-73t128-25q69 0 130 26t106 72 72 107 27 130zM326 1428q106 35 182 111t112 183L1701 640l-293-293L326 1428zm-150 444l329-82q-10-46-32-87t-55-73-73-54-87-33l-82 329zM1792 549q25-25 48-47t41-46 28-53 11-67q0-43-16-80t-45-66-66-45-81-17q-38 0-66 10t-53 29-47 41-47 48l293 293z" />
   </svg>
 ));
+
+/* All collocated stories used in react-examples storybook require a title */
+export default {
+  title: 'Dummy',
```

Comment:
There are actually no stories in the file, so there will be no 'Dummy' in the storybook.

The question is - these are components used solely in stories, where should we locate them? How should be the file named? Also see `ColorRamp.tsx` in this PR - the same problem.

Reply:
Or perhaps we could keep this without the title and ignore entries without default export in preview.js? (https://github.com/microsoft/fluentui/pull/18238#discussion_r634436049)

Reply:
this is solely a helper/utility file for anything related to storybook in particular package, thus I don't recommend to provide default export with title.

**👉 Action Required:** please remove this default

> ` All collocated stories used in react-examples storybook require a title`

- are you sure about this ?

> The question is - these are components used solely in stories, where should we locate them? How should be the file named?

---

_I'd suggest following:_
-  keep anything related to storybook within `.stories.tsx?` file pattern as is, because in future we gonna filter them out for package production builds via per environment TS configs (solution files)*🧵
  - thus this will prevent anyone to import storybook related things to production package by accident (TS error)
- if there will be way to many helpers for particular package, I'd extract them to sibling private package (to make enforce domain boundaries)
ex:  
```
|- packages/
|- |- react-menu/
|- |- react-menu-sb-utils/

--------------       ------------------------
| react-menu |  --> | react-menu-sb-helpers |
--------------       ------------------------
```

***🧵More about via per environment TS configs (solution files)**
  -  we use this approach for  newly added `/tools` (https://github.com/microsoft/fluentui/pull/18194/files#diff-a782d0e702dca71f09d0fbd5e83fc8511e70d7327ac18f95d16fd03f5c4795f0)
  - for production build tsconfig.lib.json will be used with something like following:
  ```
  "exclude": [
    "**/*.spec.ts",
    "**/*.spec.tsx",
    "**/*.stories.ts",
    "**/*.stories.js",
    "**/*.stories.jsx",
    "**/*.stories.tsx"
  ],
  "include": ["**/*.js", "**/*.jsx", "**/*.ts", "**/*.tsx"]
  ```

Reply:
>All collocated stories used in react-examples storybook require a title
>
>are you sure about this ?

For sorting, preview.js uses the title prop regardless if there's is one.
I assume that's what @miroslavstastny  is mentioning.

Reply:
OK, this makes sense. So I renamed the file back to `tmp-icons.stories.tsx` and removed the default export.

Reply:
But I still filter out the `.stories` without the default export in `preview.js`.
Are you ok with the requirement (for now)?